### PR TITLE
fix(curriculum): add examples in dice game step 14

### LINKED ISSUE #56674: js beta dice game step 14 should give a more elaborate example
### Describe the Issue

The number of times someone on the forum has posted asking for help with step 14 of the dice game (js beta project) is not small. Many of these people have the same mistake of assuming that the checkForStraights function need only to check for consecutive numbers in the dice array. (and they are always surprised to learn that this is not the case). I don't know if this is because they are not imagining that the diceValuesArr contains an unordered list of dice rolls or not. Whatever the reason, I think to mitigate this issue, a more elaborate example can be added to the step to do away with that assumption once and for all.
Specifically instead of:

>A small straight is when four of the dice have consecutive values in any order (Ex. 1234) resulting in a score of 30 points. A large straight is when all five dice have consecutive values in any order (Ex. 12345) resulting in a score of 40 points.

We can have this:

A small straight is when four of the dice have consecutive values in any order (Ex. in a roll of 44123, we have 1234) resulting in a score of 30 points. A large straight is when all five dice have consecutive values in any order (Ex. in a roll of 31245, we have 12345) resulting in a score of 40 points.

It makes the text longer, but it may save some confusion. Please see a sample list of forum posts all sharing this confusion below.

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/review-algorithmic-thinking-by-building-a-dice-game/step-14

### Your code

N/A

### Expected behavior

Change the description to give a more elaborate example illustrating the unordered array as a starting point.

### Screenshots

_No response_

### System

N/A

### Additional context

https://forum.freecodecamp.org/t/review-algorithmic-thinking-by-building-a-dice-game-step-14/710121/4
https://forum.freecodecamp.org/t/review-algorithmic-thinking-by-building-a-dice-game-step-14/708650/11
https://forum.freecodecamp.org/t/review-algorithmic-thinking-by-building-a-dice-game-step-14/706200/5
https://forum.freecodecamp.org/t/review-algorithmic-thinking-by-building-a-dice-game-step-14/707520/4
https://forum.freecodecamp.org/t/review-algorithmic-thinking-by-building-a-dice-game-step-14/712629/2
https://forum.freecodecamp.org/t/review-algorithmic-thinking-by-building-a-dice-game-step-14/713961/12
https://forum.freecodecamp.org/t/review-algorithmic-thinking-by-building-a-dice-game-step-14/716221

### PR DESCRIPTION
**Pull request by kamalsinghb**
Checklist:

- [x ] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [ x] My pull request targets the `main` branch of freeCodeCamp.
- [ x] I have tested these changes either locally on my machine, or GitPod.

Closes #56674

image

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-algorithmic-thinking-by-building-a-dice-game/657e230500602983e01fff6e.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: step-14
 
 # --description--
 
-For the last portion of the game, you will need to create an algorithm that checks for the presence of a straight. A small straight is when four of the dice have consecutive values in any order (`Ex. 1234`) resulting in a score of `30` points. A large straight is when all five dice have consecutive values in any order (`Ex. 12345`) resulting in a score of `40` points.
+For the last portion of the game, you will need to create an algorithm that checks for the presence of a straight. A small straight is when four of the dice have consecutive values in any order (`Ex. in a roll of 41423, we have 1234`) resulting in a score of `30` points. A large straight is when all five dice have consecutive values in any order (`Ex. in a roll of 35124, we have 12345`) resulting in a score of `40` points.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-algorithmic-thinking-by-building-a-dice-game/657e230500602983e01fff6e.md:10]:
```suggestion
For the last portion of the game, you will need to create an algorithm that checks for the presence of a straight. A small straight is when four of the dice have consecutive values in any order (Ex. in a roll of `41423`, we have `1234`) resulting in a score of `30` points. A large straight is when all five dice have consecutive values in any order (Ex. in a roll of `35124`, we have `12345`) resulting in a score of `40` points.
```

what's in backticks can't be translated

Comment (kamalxdev) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-algorithmic-thinking-by-building-a-dice-game/657e230500602983e01fff6e.md]: done

**Comment by kamalxdev** [comment]
Changes have been committed.

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Review by gikf — APPROVED** [review]

**Pull request merged**